### PR TITLE
Added some new subjects

### PR DESCRIPTION
**Pull request by DavideCristoni**

### CODE REVIEW

**Review by boozec — APPROVED** [review]
LGTM but we're also considering to use Telegram channels instead of so many groups. Thank you

**Comment by foxyseta** [comment]
So are intentionally leaving some of the "chat" fields empty for now? If so, people should be very careful and remember to fill them up us soon as channels/groupchats or whatever are ready.

**Comment by DavideCristoni** [comment]
@foxyseta some of the fields were already empty. For every course that I've added I also created the tg group. I will also create the other 2 missing.

**Comment by foxyseta** [comment]
Noce! Want me to accept the PR now or after you are done with the groupchats?

**Comment by boozec** [comment]
> use Telegram channels instead of so many groups

Sorry, mate. I meant "group topics" 😅

File: teachings.json [hunk 1]
```diff
@@ -900,10 +900,37 @@
...
+    "chat": "https://t.me/+9wTytT_D4Lg1MjFk",
     "website": "479049",
     "professors": [
       "giuseppe.lisanti"
     ]
+  },
+  {
+    "name": "Artificial Intelligence, Blockchain e criptovalute nello sviluppo software",
+    "url": "artificial-intelligence-blockchain-e-criptovalute-nello-sviluppo-software",
+    "chat": "@AIBC_unibo",
```
Comment (foxyseta) [inline teachings.json:912]: I am pretty sure a full url is required. As groups on different instant messaging apps are supported.

**Review by foxyseta — CHANGES_REQUESTED** [review]

File: teachings.json (hunk 1 above)
Comment (DavideCristoni) [reply teachings.json:912]: > I am pretty sure a full url is required. As groups on different instant messaging apps are supported.

Sorry! My mistake! I will patch it up soon. Then, if there are no problems, I think we could pass this PR so at least we have some new pages for courses that hust started.

Comment (DavideCristoni) [reply teachings.json:912]: > I am pretty sure a full url is required. As groups on different instant messaging apps are supported.

Sorry! My mistake! I will patch it up soon. Then, if there are no problems, I think we could pass this PR so at least we have some new pages for courses that just started.

Comment (foxyseta) [reply teachings.json:912]: Yes, will merge is as soon as the path is pushed. After that happens, we must also merge dependabot's PR to update this dependency (csunibo/config) within the other projects (e.g., dynamik and informabot)

**Comment by foxyseta** [comment]
> > use Telegram channels instead of so many groups
> 
> Sorry, mate. I meant "group topics" 😅

Group topics still have topic-specific links, so I'd say we'll be able to simply update the links when we'll switch to those. Feel free to open an issue in this repository. Technically, IPIM is already using them. Are their teachings in teachings.json using topics-specific links already?

**Review by foxyseta — APPROVED** [review]

**Pull request merged**